app/javascript/packs/validateSignup.js

```diff
@@ -16,30 +24,33 @@ document.addEventListener('DOMContentLoaded', () => {
       presence: true,
       email: true,
       length: { minimum: 1 }
-    },
+    }
+  }
+  // Fields 'password' and 'password_confirmation' are optional (can be disabled)
+  const isPasswordRequired = document.querySelectorAll('input[type="password"]').length > 0
```

Comment:
I agree, but the problem is that we are using a liquid template that won't be updated.

Reply:
@josemigallas , by "risky", do you mean unreliable or insecure?

Reply:
It's unreliable, but anyway, the problem is that we can't update the liquid template (they are copied from database in the moment of the creation of the account), then there is no way to pass the value from backend (we should ask customers to manually edit templates, and I think we should avoid that).
Besides, I think this feature of disabling password should be removed, because in the UI there is no option to disable it, there is only the option to add fields, but not to remove them. And there is nothing in the docs relatedto disabling password.
I think is a legacy feature that was removed, but some pieces of code were not removed.
But who knows that?

Reply:
> I agree, but the problem is that we are using a liquid template that won't be updated.

Ah, yes, you're right!

So the real problem is the user could modify their template (renaming the password input), unknowingly removing validation... but that's how the dev portal works anyway.